core/src/test/java/org/apache/iceberg/rest/responses/TestLoadTableResponseParser.java

```diff
@@ -58,71 +59,73 @@ public void missingFields() {
 
   @Test
   public void roundTripSerde() {
-    String uuid = "386b9f01-002b-4d8c-b77f-42c3fd3b7c9b";
-    TableMetadata metadata =
-        TableMetadata.buildFromEmpty()
-            .assignUUID(uuid)
-            .setLocation("location")
-            .setCurrentSchema(
-                new Schema(Types.NestedField.required(1, "x", Types.LongType.get())), 1)
-            .addPartitionSpec(PartitionSpec.unpartitioned())
-            .addSortOrder(SortOrder.unsorted())
-            .discardChanges()
-            .withMetadataLocation("metadata-location")
-            .build();
+    for (int formatVersion : Arrays.asList(2, 3)) {
```

Comment:
I don't want to run all the tests for each format, let me know if there is a better way to do this.

Reply:
it's probably good to annotate the test with `@ParameterizedTest` and pass the format version as an argument. This makes it also easier to debug in case it fails with one format version but not with the other

Reply:
Exactly what I was looking for, thanks!